README.md

```diff
@@ -1,40 +1,58 @@
-Prometheus exporter that scrapes meta information about a running ceph cluster. All the information gathered from the cluster is done by interacting with the monitors using an appropriate wrapper over `rados_mon_command()`. Hence, no additional setup is necessary other than having a working ceph cluster.
+
+A Prometheus exporter that scrapes meta information about a running Ceph
+cluster. All the information gathered from the cluster is done by interacting
+with the monitors using an appropriate wrapper over
+`rados_mon_command()`. Hence, no additional setup is necessary other than
+having a working Ceph cluster.
 
 ## Dependencies
 
-You should ideally run this exporter from the client that can talk to
-Ceph. Like any other ceph client it needs the following files to run
+You should ideally run this exporter from the client that can talk to the Ceph
+cluster. Like any other Ceph client, it needs the following files to run
 correctly.
 
- * `ceph.conf` containing your ceph configuration.
- * `ceph.<user>.keyring` in order to authenticate to your cluster.
+* `ceph.conf` containing your Ceph configuration.
+* `ceph.<user>.keyring` in order to authenticate to your Ceph cluster.
 
-Ceph exporter will automatically pick those up if they are present in
+The `ceph_exporter` will automatically pick those up if they are present in
 any of the [default
-locations](http://docs.ceph.com/docs/master/rados/configuration/ceph-conf/#the-configuration-file). Otherwise you will need to provide the configuration manually using `--ceph.config` flag.
+locations](http://docs.ceph.com/docs/master/rados/configuration/ceph-conf/#the-configuration-file). Otherwise
+you will need to provide the configuration manually using environment
+variables:
+
+* `CEPH_CLUSTER`: cluster's name (default `ceph`)
+* `CEPH_CONFIG`: configuration file that a Ceph client uses to connect to
+  the cluster (default `/etc/ceph/ceph.conf`)
+* `CEPH_USER`: a Ceph client user used to connect to the cluster (default
+  `admin`)
 
-We use Ceph's [official Golang client](https://github.com/ceph/go-ceph) to run commands on the cluster.
+We use Ceph's [official Golang client](https://github.com/ceph/go-ceph) to run
+commands on the cluster.
 
-Ceph exporter is tested only on Ceph's Hammer(v0.94) and Jewel(v10.2) releases. It might not work as expected with older or non-LTS versions of Ceph.
+The `cepp_exporter` is tested only on Ceph Nautilus releases. It might not
```

Comment:
s/cepp/ceph/

Also, might want to specify `This ceph_exporter branch is tested ...`, just to make it clear that the user should look at other branches for other releases.

Reply:
Sure, updated.

Reply:
:+1: